packages/tests/server/react/useContext.test.tsx

```diff
@@ -375,7 +375,7 @@ test('setData', async () => {
 test('setInfiniteData', async () => {
   const { proxy, App } = ctx;
   function MyComponent() {
-    const listPosts = proxy.post.list.useQuery({}, { enabled: false });
+    const listPosts = proxy.post.list.useInfiniteQuery({}, { enabled: false });
```

Comment:
This was previously a bit of a bug in the tests that went unnoticed ... Before we were updating a `useQuery` with an `infinite` data structure ... With the new distinction this test broke! 

I've changed this over and fixed.